src/frequenz/sdk/timeseries/pv_pool/_pv_pool.py

```diff
@@ -0,0 +1,201 @@
+# License: MIT
+# Copyright © 2024 Frequenz Energy-as-a-Service GmbH
+
+"""Interactions with pools of PV inverters."""
+
+import asyncio
+import typing
+import uuid
+from collections import abc
+from datetime import timedelta
+
+from ..._internal._channels import ReceiverFetcher
+from ...actor import _power_managing
+from ...timeseries import Bounds
+from .._base_types import SystemBounds
+from .._quantities import Power
+from ..formula_engine import FormulaEngine
+from ..formula_engine._formula_generators import FormulaGeneratorConfig, PVPowerFormula
+from ._pv_pool_reference_store import PVPoolReferenceStore
+from ._result_types import PVPoolReport
+
+
+class PVPoolError(Exception):
+    """An error that occurred in any of the PVPool methods."""
+
+
+class PVPool:
+    """An interface for interaction with pools of PV inverters.
+
+    !!! note
+        `PVPool` instances are not meant to be created directly by users. Use the
+        [`microgrid.pv_pool`][frequenz.sdk.microgrid.pv_pool] method for creating
+        `PVPool` instances.
```

Comment:
The class documentation is what I see most of the time through editor tools.  Can I remove the constructor note instead?

Also yes, it is there twice in the other pools as well.

Reply:
Mmm, for me it makes more sense to keep it in the constructor, if you want to create one you should read the constructor documentation, while if you just want to use it, you probably will read the class documentation but you don't care about that note because you don't plan to instantiate it anyway, so it is noisy.

But if you say your editor shows the class docs instead when you call the constructor, then I guess we can keep it in both.

Reply:
I think you're right, that note is not necessary in the context of just the class, and I think if I try to call the constructor I will get the constructor docs.

I've added a commit to remove them for all pools.